src/drawing.rs

```diff
@@ -541,6 +542,47 @@ impl Drawing {
     pub fn poll_event(&mut self) -> Option<Event> {
         self.window.borrow_mut().poll_event()
     }
+
+    /// Saves the current drawings in SVG format at the location specified by `path`.
+    ///
+    /// ```rust
+    /// use turtle::{Turtle, Color};
+    ///
+    /// fn main() {
+    ///     let mut turtle = Turtle::new();
+    ///     turtle.drawing_mut().set_background_color("pink");
+    ///
+    ///     for i in 0..36 {
+    ///         let base_color: Color = if i % 2 == 0 {
+    ///             "red".into()
+    ///         } else {
+    ///             "white".into()
+    ///         };
+    ///         turtle.set_fill_color(base_color.with_alpha(1.0 - i as f64 / 54.0));
+    ///         turtle.begin_fill();
+    ///         square(&mut turtle);
+    ///         turtle.end_fill();
+    ///         turtle.right(10.0);
+    ///     }
+    ///    turtle.hide();
+    ///    turtle.drawing().save_svg("squares.svg");
```

Comment:
Looks like these lines are slightly de-indented. Could you make sure this example has 4-space indents all throughout? I think this is probably just a copy/paste reformatting issue. :)